### PR TITLE
feat: added Navbar submenus and drawer to hamburger 

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
image
image
image

### CODE REVIEW

File: component/navbar/navbar.css [hunk 1]
```diff
@@ -19,6 +19,54 @@
...
+    background-color: var(--white);
+    text-align: center;
+    padding-inline: 10px 10px;
+    color: var(--black);
+}
+.submenu-items{
+    background-color: var(--white);
+    text-align: center;
+    padding-inline: 10px 10px;
+    color: var(--black);
```
Comment (tarunsankhla) [inline component/navbar/navbar.css:68]: I think 1 10px.will do the work

File: component/navbar/navbar.html [hunk 2]
```diff
@@ -144,6 +145,64 @@ <h1>Navbar</h1>
...
+        style="width: 1024px; height: 532px; border:0; transform: scale(1); overflow:hidden;"
+        sandbox="allow-scripts allow-same-origin">
+      </iframe>
+
+      <h3>Navbar with Hamburger/Drawer</h3>
+            <div class="navbar-container">
+                <div class="navbar-component">
+                    <div class="hamburger">
```
Comment (tarunsankhla) [inline component/navbar/navbar.html:179]: Add comments in your html code

**Review by tarunsankhla — APPROVED** [review]
Looks good

File: component/navbar/navbar.html [hunk 3]
```diff
@@ -144,6 +145,64 @@ <h1>Navbar</h1>
...
+  style="width: 1024px; height: 550px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```
Comment (deekshithmd) [inline component/navbar/navbar.html:204]: try to align code  properly, it will give good readability, rest everything looks good

**Review by deekshithmd — APPROVED** [review]

File: component/navbar/navbar.css (hunk 1 above)
Comment (dev-ashishsethi) [reply component/navbar/navbar.css:68]: I wanted padding at start and end

**Pull request merged**